tests/conftest.py

```diff
@@ -62,34 +62,58 @@ def s2p_stat_fixture(tmp_path: Path, request) -> Tuple[Path, dict]:
 
 @pytest.fixture()
 def motion_correction_fixture(tmp_path: Path, request) -> Tuple[Path, dict]:
-    """Fixture that allows parameterized mock motion correction video files
-    to be generated
+    """Fixture that allows parameterized mock motion correction files (*.csv)
+    files to be generated.
+
+    Old-style motion files do *not* contain a header row (yikes D:)
+    You just have to know that they contain 9 columns with the following names:
+    ["index", "x", "y", "a", "b", "c", "d", "e", "f"]
+
+    New-style motion files may have more or fewer than 9 columns but will have
+    a header row (which contains at least one alphabetical character [a-zA-Z]).
     """
+    # Range of mock motion correction values to generate
     abs_value_bound = request.param.get("abs_value_bound", 5.0)
-    motion_correction_rows = request.param.get("motion_correction_rows", 15)
-    included_values_x = request.param.get("included_values_x", [])
-    included_values_y = request.param.get("included_values_y", [])
+    default_col_names = ["index", "x", "y", "a", "b", "c", "d", "e", "f"]
+    col_names = request.param.get("col_names", default_col_names)
```

Comment:
do you still need the col_names and the default_col_names in this fixture with the other functions removed?

Reply:
I'm okay with the fixture still having the ability to create deprecated versions of the motion correction file. The default is for it not to.